API/routes/userRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+
+const router = express.Router();
+const userController = require('../controllers/userController');
+const socialLogInToken = require('../utils/auth');
```

Comment:
외부에서 모듈을 가지고 오는 코드를 항상 맨 위쪽으로 가게 해주세요.
```suggestion
const express = require('express');

const userController = require('../controllers/userController');
const socialLogInToken = require('../utils/auth');

const router = express.Router();
```